dvc/output/local.py

```diff
@@ -16,82 +15,52 @@
 
 class OutputLOCAL(OutputBase):
     REMOTE = RemoteLOCAL
+    sep = os.sep
 
-    def __init__(
-        self,
-        stage,
-        path,
-        info=None,
-        remote=None,
-        cache=True,
-        metric=False,
-        persist=False,
-        tags=None,
-    ):
-        super(OutputLOCAL, self).__init__(
-            stage,
-            path,
-            info,
-            remote=remote,
-            cache=cache,
-            metric=metric,
-            persist=persist,
-            tags=tags,
-        )
-        if remote:
-            p = os.path.join(
-                remote.prefix, urlparse(self.url).path.lstrip("/")
-            )
+    def _parse_path(self, remote, path):
+        parsed = urlparse(path)
+        if parsed.scheme == "remote":
+            p = remote.path_info / parsed.path.lstrip("/")
         else:
-            p = path
-
-        if not os.path.isabs(p):
-            p = self.remote.to_ospath(p)
-            p = os.path.join(stage.wdir, p)
-        p = os.path.abspath(os.path.normpath(p))
+            p = self.REMOTE.path_cls.from_posix(path)
```

Comment:
ok, so, is path always posix here?

Reply:
The thing is output can't know whether it got path from `.dvc` file, which is always posix or cmd line, which could be windows style. So this should work whatever style path you passed.

Reply:
I would check that.

Reply:
Made code more explicit. BTW, it also has #2075 issue. Made it behave the old way until that resolved.